### ISSUE TITLE
Full Experiment Management

### ISSUE THREAD

**Issue by michael-butler1994** [huntallcott/PhoneAddiction#17]
Started this thread to discuss management of the Full Experiment. I’ve reformatted the notes from yesterday’s meeting. It’s now a checklist of things we would need to get done to run the full experiment this semester. Please take a look [here](https://docs.google.com/document/d/14rvUWxbH7xR8MlNCIUGnnSr33aTU1e_bXf7N_cccHSo/edit).  I realize that the notes contain a bit of content discussed on other threads, but I thought it would be helpful to have this checklist in a central location on Github. Perhaps we can continue discussing the specifics of each issue on their individual threads. Let me know if ya'll would prefer a different workflow.

**Comment by lenasong** [comment]
@michael-butler1994 thanks for the checklist. 

Thinking about @gentzkow's comment about transcript and whether we will detect any effect, I wonder if it's worthwhile to step back for a moment and consider other objective outcomes (and minimize selective attrition if possible).

Here are a couple of ideas we've discussed previously:

1. task performance after endline 
There are several psychology studies that show the mere presence of a phone affects social interactions and performance in various tasks, especially for more complex tasks - see for example, Przybylski & Weinstein (2013), Thornton et al (2014). (@huntallcott did you have a different study in mind in your comments [here](https://docs.google.com/document/d/1JN-EoCgx4FZLcAk9Wp8bXGrrm4kHsR7eyhF_V_TixnY/edit) when this idea was brought up?)

These studies vary whether or not a phone is present during the study (not even the participant's phone), so I'm not convinced by the results. However, we could have a similar set-up to see whether our treatment vs. control groups are more/less prone to distractions? This could be done in-person or via online assignments - depending on how we recruit. 

2. participant daily activities via either participant diary or frequent short check-in surveys 

Obviously there will be attrition for both of these outcomes but we could phrase the task performance part as a separate study and invite everyone we recruited to participate.

**Comment by lenasong** [comment]
@michael-butler1994 @huntallcott @gentzkow thanks for the comments in #18. 

For the next step of incorporating these into our design and writing out exact questions, I made a list in the [experiment design](https://docs.google.com/document/d/1WH9EAjsdJhLjE0jzieUN4UgSiaR3ug8kdIgyjVhBfGw/edit#) document of survey instruments we need to add.

@michael-butler1994 it'd be great if you can take the lead on this. The topics are listed on page 3 of the experiment design and details for each topic can be found in the [outcomes list](https://docs.google.com/document/d/1v3ORShPNL7wyM_Db71ZYpyVJguEpft-HaDC41V-_VcU/edit#). I've also included references to our lit review and lists of possible survey instruments that would be helpful for each topic. 

As discussed over the phone, let's aim to finish this in a week or so by the Monday following. This would give us a sense of the outcome variables to test in pilot3 and get us in a good position for IRB.  

Have a great long weekend!

**Comment by michael-butler1994** [comment]
@lenasong @gentzkow @huntallcott 
I've drafted a [baseline survey instrument](https://docs.google.com/document/d/1zb_x73LLjkU-TqPGcUHrS6ULYx-dPwCxV23nkPo4yCA/edit#heading=h.3c0zj3dbbljw) covering everything except Belief Elicitation and the MPL. I also did not include survey questions we may ask participants _during_ the baseline (e.g. keeping a sleep diary). It would be great to get some feedback! The [experimental design](https://docs.google.com/document/d/1WH9EAjsdJhLjE0jzieUN4UgSiaR3ug8kdIgyjVhBfGw/edit#) doc has several additional survey questions I considered including but that did not make the initial cut.

**Comment by lenasong** [comment]
@michael-butler1994 thanks, this is awesome! I've made some edits and added a few comments. I also highlighted some questions that I think would be helpful to test out in the next pilot. @gentzkow @huntallcott take a look and let us know what you think.

**Comment by huntallcott** [comment]
@michael-butler1994 thanks for putting this together. I just left you a bunch of comments.

@gentzkow should also review as soon as he gets to this.

One general theme is that I think we want to have in mind how this will map into a pre-analysis plan. There are only three types of questions:
* Outcomes
* Basic demographics for weighting and/or stratification
* Moderators
And the latter two categories are going to be pretty limited, so there is a high bar.
Some of the questions I suggested dropping didn't seem to fit in any of these categories.

Then for outcomes, we want to be able to block them into well-defined groups to create our indices (just as in the FB paper). And I think we would similarly put those questions together in a block on the survey so it's easier to answer. So I left some suggestions on that theme as well.

Can you take another pass?

**Comment by michael-butler1994** [comment]
@huntallcott thanks for all the feedback! I implemented nearly all of your edits/suggestions as track changes. I replied to your comments when I hesitated to implement your suggestions.

**Comment by huntallcott** [comment]
* @michael-butler1994 many thanks.
* @lenasong: can you take a stab at the hypothetical MPL text and belief elicitation text? (Remembering that we are doing hypothetical payments at baseline, but we want the text to parallel the actual payment text that we'll use in the two midlines.)
* @gentzkow can you take a look at the survey questions when you have a moment?

* I think a key next step here is to write up the beginning of a PAP that outlines the questions we're asking and how they fit together into indices/groups. This will help us to make sure that the questions that @michael-butler1994 has drafted all fit together super-nicely. I am happy to get it started. Unfortunately I can't do it quite yet but will try to get to it soon.

**Comment by lenasong** [comment]
@michael-butler1994 @huntallcott @gentzkow I added text for MPL and belief elicitation in our [survey instrument ](https://docs.google.com/document/d/1zb_x73LLjkU-TqPGcUHrS6ULYx-dPwCxV23nkPo4yCA/edit?ts=5c7958ba#)file. It's organized by topic and includes text variations for both baseline and midlines. I also drafted in-app text to outline how Phone Dashboard would look.

Take a look? I left some questions about incentivizing belief elicitation and app design.

**Comment by huntallcott** [comment]
@lenasong!
Thanks for your work on the survey!
I left some comments and suggestions. In addition:
1. Will we observe the list of all apps the user has, from Pennyworth? That would be nice because then we have a non-self-report list of whether they have other blocking functionality. 
2. @gentzkow: I realized I’m a bit confused about how the Costly Snooze Button will work, given that we decided that these blockers are separate from the Screen Time Reduction Incentive that we’ve been modeling in the identification document, and so we're not paying people to use these blockers. So for example, some people who have the Costly Snooze Button will not be in the Screen Time Reduction Incentive group, so their only payment is the participation incentive. Is the idea that we’ll reduce your participation payment by $1 (or whatever) every time you hit the snooze button? Does that possibly generate biased attrition?

**Comment by gentzkow** [comment]
@lenasong @michael-butler1994 @huntallcott Apologies for the *very* slow reply here and on some other threads. Thanks for the great work on this so far.

I've read through the [survey instrument](https://docs.google.com/document/d/1zb_x73LLjkU-TqPGcUHrS6ULYx-dPwCxV23nkPo4yCA/edit?ts=5c7958ba#) and re-read the [experimental design doc](https://docs.google.com/document/d/1WH9EAjsdJhLjE0jzieUN4UgSiaR3ug8kdIgyjVhBfGw/edit#). They look great in general. 

Minor: I have to say I found it a little hard to follow in the survey doc what questions were being asked in baseline / midlines / endline and in what order. In some cases it's indicated which survey something is in; I wasn't sure if things with no indication are baseline only, on all surveys, or something else. I got particularly lost at the end once we got to belief elicitation, WTA, etc. At some point I think it would be helpful to expand this out into separate docs for those surveys individually. For now, apologies if any of my comments reflect misunderstanding.

I've added various comments to the Google doc. I'm afraid a lot of them are questions rather than suggestions -- apologies if some of these are things we've discussed and I've missed.

I very much like @huntallcott's typology of outcomes / demographics / moderators. It would be great if there is a way to summarize somewhere which questions we're thinking of as which. 

I'm thinking of the following as outcomes:
* Substitution / time use
* Academics
* Sleep
* SWB
* Adult behavior
* Feelings about phone use

Questions I wasn't sure whether they were outcomes or potential moderators:
* GPA
* Class schedule
* Study habits
* Social media / communication
* Questions about phone use <-> sleep like how long until I check my phone in the morning

**Comment by gentzkow** [comment]
@huntallcott On the snooze question [here](https://github.com/huntallcott/PhoneAddiction/issues/17#issuecomment-469885145), I had initially imagined we would give this only to people in the Screen Time Reduction Incentive group so hitting the snooze button would be similar to getting charge extra minutes of use. But I do like the idea of cross-randomizing this so we get some people with the blockers only and no incentive. I would have thought in this case we'd just give people who land in that group an extra payment that they can use as a "budget" to spend on snooze. So e.g. we say you get $20 minus X per snooze where X is a number you can choose from {0, $0.50, $1, etc.}. Presumably if you exhaust the budget subsequent snoozes just become free.

**Comment by huntallcott** [comment]
RE: costly snooze button. Great, the $20 budget sounds great. Incidentally, I think we should look at this in the pilot and see if how we can use for estimation and/or if it's really interesting descriptively. If this is still just a descriptive treatment (i.e. one that is neat but we're not using it to estimate betahat and beta) by the time we get to the final experiment, I might vote we should kill it because it adds a bit of complexity.

RE: outcomes and moderators: @lenasong I was wondering if you could look back at the experimental design doc and outcome variable list to help us get organized here. I think we want to have a concise page there that gives:
* the list of specific demographic variables that we'll use as descriptive stats and balancing
* the list of variables we'll use for stratification. (these should be approx 10-40 cells split on our key moderators)
* the name of each of our 5-10 families of outcomes
* within each family, the shortened name of each variable that we'll use in qualtrics/stata
* within each family, the 1-3 key moderators that we'll pre-specify.

This one-pager will contain ALL variables we want.
Then we can go back and make sure that *all* variables in the baseline survey instrument are either outcomes, demographics, or instruments, or a *limited* number of exploratory pilot variables.

That for me would be really helpful. @lenasong @gentzkow @michael-butler1994 if you guys think that's a good idea, then @lenasong would you be willing to do that?

**Comment by lenasong** [comment]
@huntallcott @michael-butler1994 @gentzkow thanks for great feedback.

Yes I can work on the summary of variables. As part of this, I'll also clean up and edit the [survey instruments](https://docs.google.com/document/d/1zb_x73LLjkU-TqPGcUHrS6ULYx-dPwCxV23nkPo4yCA/edit?ts=5c7958ba#) this weekend. @michael-butler1994 @huntallcott if you have time before that, can you take a look at our new comments?

**Comment by lenasong** [comment]
This is going to be a long post - apologies in advance. 

1. I've drafted a [list of variables](https://docs.google.com/document/d/1c3Qikp2eb_-luDiPZbQQ7qaOJQFdJftbVD_x2EI663U/edit#heading=h.jrm7qskrg35v) that summarizes the families of variables. I'm erring on the side of completeness - feel free to make any edits and suggestions.

2. @michael-butler1994 @gentzkow @huntallcott I reviewed the comments in [survey instrument](https://docs.google.com/document/d/1zb_x73LLjkU-TqPGcUHrS6ULYx-dPwCxV23nkPo4yCA/edit?ts=5c7958ba#) and made edits/comments. Take a look and make sure you agree with the content? The questions/comments I want to highlight are:

- Unofficial transcript request: we need to ask for transcript weeks after endline when grades are posted. Should we do it separately as a follow-up survey for the baseline too? I also wonder how much more value we get from this vs. a self-reported GPA.
- Language used in introduction to payments: is there a reason why we prefer $q/hour vs. $p/average hour of reduction? Our belief elicitation are based on daily averages so the former might be confusing 
- Point of price non-linearity: have we converged to a number here (e.g. participant's baseline usage + buffer)?
- Language for what this point of non-linearity is: do we want to be explicit that this point is set to be the baseline usage? Alternatively we could just set this value without saying where it comes from (in the case buffer > 0)  
- rewards for accurate predictions: AR(2018) finds "little evidence that participants are using the prediction to manipulate the incentives of their future selves" and the size of the bonus didn't matter. They found "many participants prefer to exactly match their previous predictions" even when the rewards are too small to be explained by monetary preferences. This anchoring effect wasn't incorporated in the analysis and I think is less relevant in our setting. So given the findings of their study, we can randomize people into a low reward of $0.25 and high reward of $8.25 (the lowest and highest on their 14 rewards with total average payment of $110 for the study).
- @huntallcott yes we can observe whether people are using other digital wellbeing apps. Small point - we don't know whether they are using them to impose limit 
- Will organize into baseline/midline/endline in the next iteration with file outlined below 
 
3. since we have several data sources, I started putting together a [detailed instrument file](https://docs.google.com/spreadsheets/d/1DstkwyCnPrRP9oG4NWp3YJ9id7SoNggK-ncKwrH-uvs/edit#gid=0) that includes _all_ questions from surveys, Phone Dashboard, Web Historian and emails, as well as how each question translates to a variable (in the case of indices). If you agree this is useful, I can move the survey instrument content here after we finalize the content above.

4. @gentzkow it'd helpful to get an undergrad RA to review our survey questions - thanks for offering! Depending on when they are available, I can either 1) send you an excel spreadsheet with survey questions this week or 2) send you a link to a Qualtrics survey later next week. Would be great to have a representative young person to check 1) whether the questions make sense to them 2) if we are missing anything important for how young college students interact with their smartphones

All materials are saved under GitHub\PhoneAddiction\docs\design\experimental design

**Comment by huntallcott** [comment]
@lenasong this is great work on the list of variables and groupings. I left a bunch of comments in that doc. Can you take a look and implement?
One other comment: I don’t think we should have the detailed instrument file. Let’s just have your list of variables and the survey instrument docs. It seems like the detailed instrument file just duplicates the info in those other google docs. In the survey instrument google doc, please do continue to include the links to the sources of the questions.
I have to run now but will post on the other stuff later!

**Comment by huntallcott** [comment]
@lenasong, I continue to apologize for my delay. 
Point 2 of your post above had a bunch of really interesting bullet point questions. Below, I respond to those bullets, in order.
* Unofficial transcript request. I think we can do this at baseline, and then again at post-endline. In terms of self-reported GPA: the outcome variable is current-semester GPA. Most people don’t know that, so they’d have to type in their current-semester grades. I don’t think it’s that hard to download your transcript and give it to us, compared to manually typing in your grades for the semester. 
* I see. So we’d say $20 for every hour that you reduce your average phone use over the next 20 days. If you think that’s better, then do it!
* For the pilot, what if we round up to the next hour? So if your baseline average usage is 2.9 hours, your subsidy baseline = 3 hours? That generates a bit of cushion and also makes it easy to explain. (The cutoff may also generate some identification.)
* No. I think I would just say what their incentive is, we don’t need to say where their subsidy baseline came from. 
* OK!
* Great
* Thanks

**Comment by gentzkow** [comment]
Thanks @lenasong! I also apologize for the slow reply here.

I've looked over the survey instrument and the replies / changes based on my earlier comments. All look good. Only two things to flag:

* I continue to wonder about q11 "Aside from Facebook, ESPN, Instagram etc (list will become more comprehensive), what websites do you browse when you procrastinate?" It seems to me hard to answer and I'm not sure how useful it will be. My guess is the vast majority of procrastinating people do is not browsing websites but using apps.

* I didn't see a reply to my comment on Social media/communication q2 "I care a lot about how many people respond to my social media content." I continue to think it might be worth investing in expanding this to maybe several questions that get at different aspects of this -- do you check frequently to see how many likes etc. you have, do you compare your stats to others, do you sometimes feel anxious about this, etc. Of course it's best if there are questions like this we can borrow from past work. In any case, happy to table this for now and revisit after the pilot.

On your original bullet points [here](https://github.com/huntallcott/PhoneAddiction/issues/17#issuecomment-472257064) I am fine with all of @huntallcott's [replies](https://github.com/huntallcott/PhoneAddiction/issues/17#issuecomment-473524572).

I just sent you a separate email to put you in touch with a couple of undergrad RAs.

I'll look over the question lists next.

**Comment by gentzkow** [comment]
@lenasong I've now reviewed the [list of variables](https://docs.google.com/document/d/1c3Qikp2eb_-luDiPZbQQ7qaOJQFdJftbVD_x2EI663U/edit#heading=h.jrm7qskrg35v). This is great. I added a small number of comments.

One thing I'd just reiterate is that we want to make sure we're being systematic about which questions we ask in terms of "In the last 3 weeks." We should also think about which if any we want to ask the way we asked time use questions in the FB experiment, which was "Relative to what is usual for you, would you say in the last 3 weeks you did {less... more}"

One other misc thought: Are we sure we don't want to add a small module about news / politics to this experiment that mimics the FB one? I'd definitely want to keep this concise, but I think these issues are still interesting for the younger population. Examples of things I'd like to know are whether cutting smartphone use affects political engagement / activism and whether the news knowledge / polarization results we see are similar vs. different for this population. 

I realize this is less important in this context and I'm happy leaving it out for simplicity. I'm also happy to leave it out for the pilot and then revisit. Just wanted to flag it to make sure we're making a deliberate decision.

**Comment by lenasong** [comment]
@huntallcott @gentzkow @michael-butler1994 I've reviewed and responded to everyone's edits and comments within the Google docs - let me know if I missed anything. 

@huntallcott rounding up of baseline usage sounds great. I will link the [list of variables](https://docs.google.com/document/d/1c3Qikp2eb_-luDiPZbQQ7qaOJQFdJftbVD_x2EI663U/edit?ts=5c8ba346#heading=h.jrm7qskrg35v) to [survey instrument](https://docs.google.com/document/d/1zb_x73LLjkU-TqPGcUHrS6ULYx-dPwCxV23nkPo4yCA/edit?ts=5c8b9f4c#) with variable names once Michael creates them for the baseline in Qualtrics.

@gentzkow thanks for introducing Zane and Daniela. For your comments:
- agreed, we are dropping the question on sites used to procrastinate

- for "I care a lot about how many people respond to my social media content", we already included a few more specific questions on whether people feel jealous or anxious about interactions on social media. Happy to include additional aspects if we missed anything 

- Michael and I went through a couple more iterations on how we ask the questions, i.e. "last 3 weeks", absolute vs. relative

- I added a question on time spent on news, using the question from SocialMediaEffects that had a significant treatment effect. 

Thanks again for everyone's feedback. We are on track for pilot3! We can review these questions again when we have the baseline survey ready in Qualtrics.

**Comment by gentzkow** [comment]
Sounds great. Thanks!

**Comment by huntallcott** [comment]
Great. One comment is that it would be good to have one variable name in stata, qualtrics, and these survey instruments. So Michael please use the names I suggested so far and perhaps a similar style for the additional names.

**Comment by michael-butler1994** [comment]
@huntallcott sounds good. To maintain consistency, I may have to shorten some of your proposed variable names. stata has a 32 character limit on names and I'll need to add prefixes (e.g. "baseline_") to the key outcome variables.

**Comment by huntallcott** [comment]
@michael-butler1994 @gentzkow @lenasong one quick question,
I was just talking with Nancy Baym, the qualitative researcher at Microsoft, and she is a much bigger fan of the in-the-moment happiness questions from the FB paper than she is with retrospective questions. 
I'm also wondering if we should expect that much differential response rate to text messages between our Incentive and Control groups. 
Seems like the pilot is a good place to test that. 
Are we gathering people's phone numbers? If so, I would propose that we gather phone numbers and send people occasional text messages, e.g. 2x per week, asking happiness questions.
Sorry to be late to the party on this one. What do you think?

**Comment by lenasong** [comment]
@huntallcott from our conversations earlier it seemed that text messaging didn't work well for SocialMediaEffects? @michael-butler1994 and I are planning to ask happiness questions randomly throughout the study, sent via their preferred email address that must be answered within 24 hours. Do you expect SMS responses to be significantly better than email or very short surveys?

**Comment by huntallcott** [comment]
I think Nancy's idea is that the in-the-moment part of experience sampling is more credible than even 24-hour retrospection, although certainly both more credible than retrospection over the past 3-4 weeks. 
The text messaging worked fine for SocialMediaEffects as far as I can see. The concern we had here was about higher non-response in the Incentive group. I'm just now wondering if we should test that.
@gentzkow any thoughts?

**Comment by gentzkow** [comment]
I would say the text messaging worked *great* in SocialMediaEffects. It's one of my favorite parts of the study, and we've gotten a lot of positive feedback about it. It's one of the few components that feels like more than a regular self report (even though it's of course self reported in some sense). I think we should for sure include it in this study if we can.

A key to it working well was high response and no differential attrition. So testing that in the pilot seems like high value if it's not too late to set it up.

**Comment by lenasong** [comment]
Not too late at all! Will add this to the baseline survey.
